client/src/components/DetailView.tsx

```diff
@@ -285,7 +285,7 @@ class DetailViewWithoutI18n extends Component<Props, State> {
                       <BuildingStatsTable addr={detailAddr} />
                       <div className="card-body-timeline-link">
                         <Link
-                          to={this.props.addressPageRoutes.timeline}
+                          to={removeIndicatorSuffix(this.props.addressPageRoutes.timeline)}
```

Comment:
was there a reason for implementing `removeIndicatorSuffix` as opposed to just defaulting the timeline route to `timeline: `${prefix}/timeline/dobviolations`` ? either method works, i am just curious!